#!/usr/bin/env python
PACKAGE='dnn_detector'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name                       Type       Reconfiguration levexl            Description                       Default   Min   Max
gen.add("debug_image", bool_t, 0, "Show filtered image to debug", False)
gen.add("scaling", int_t, -1, "Scale Width and Height in Square", 180, 100, 300)
gen.add("mean_val", double_t, -1, "mean value", 127, 80, 180)
gen.add("confindence", double_t, -1, "confidence threshold", 0.5, 0, 1)

exit(gen.generate(PACKAGE, "dnn_detector", "DnnParams"))
